client/webserver/site/src/js/markets.ts

```diff
@@ -2257,7 +2258,7 @@ export default class MarketsPage extends BasePage {
 
   handleMatchNote (note: MatchNote) {
     const mord = this.metaOrders[note.orderID]
-    if (!mord) return this.refreshActiveOrders()
+    if (!mord || note.match.status === OrderUtil.NewlyMatched) return this.refreshActiveOrders()
```

Comment:
We didn't care about the new match before now, but now we do.

PS: @buck54321, is there a reason we don't consider the match `status` in `core/OrderReader.AverageRateString()`?

Reply:
Refreshing every active order in order to update two fields is a bit of overkill. A little more targeted approach here please.